renderer.js

```diff
@@ -174,3 +229,88 @@ function getSongs(e) {
             }
         });
 }
+
+
+// Suggestions
+function findMatches(wordToMatch, suggestions) {
+    return suggestions.filter(suggestion => {
+        // here we need to figure out if the city or state matches what was searched
+        const regex = new RegExp(wordToMatch, 'gi');
+        return suggestion.match(regex)
+    });
+}
+
+function displayMatches() {
+    if (this.value.length < 3) {
+        return;
+    }
+    fetch(`https://suggestqueries.google.com/complete/search?client=firefox&q=${this.value}`)
```

Comment:
For consistency sake, since you included `request` package, use request  instead of fetch. To have a consistent  code